models/rollup_economics/ethereum/rollup_economics_ethereum_l1_fees.sql

```diff
@@ -0,0 +1,136 @@
+{{ config(
+    alias = alias('l1_fees'),
+    tags = ['dunesql'],
+    partition_by = ['block_time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash'],
```

Comment:
looks like you chose to pull `hash` instead of `tx_hash` in the query. can you update here to reflect that?